frontend/app/src/components/helper/BinancePairsSelector.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <v-card v-bind="$attrs">
+    <div class="mx-4 pt-2">
+      <v-autocomplete
+        :items="allMakerts"
+        :filter="filter"
+        :search-input.sync="search"
+        multiple
+        :loading="loading"
+        :disabled="loading"
+        hide-details
+        hide-selected
+        hide-no-data
+        return-object
+        chips
+        clearable
+        :outlined="outlined"
+        :open-on-clear="false"
+        :label="label ? label : $t('blockchain_account_selector.default_label')"
+        :class="outlined ? 'blockchain-account-selector--outlined' : null"
+        item-text="address"
+        item-value="address"
+        class="blockchain-account-selector"
+        :value="selection"
+        @input="input($event)"
+      >
+        <template #selection="data">
+          <v-chip
+            :key="data"
+            v-bind="data.attrs"
+            :input-value="data.selected"
+            :click="data.select"
+            filter
+            close
+            @click:close="data.parent.selectItem(data.item)"
+          >
+            {{ data.item }}
+          </v-chip>
+        </template>
+        <template #item="data">
+          <div
+            class="blockchain-account-selector__list__item d-flex justify-space-between flex-grow-1"
+          >
+            <div class="blockchain-account-selector__list__item__address-label">
+              <v-chip :color="dark ? null : 'grey lighten-3'" filter>
+                {{ data.item }}
+              </v-chip>
+            </div>
+          </div>
+        </template>
+      </v-autocomplete>
+    </div>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Mixins, Prop } from 'vue-property-decorator';
+import AccountDisplay from '@/components/display/AccountDisplay.vue';
+import TagIcon from '@/components/tags/TagIcon.vue';
+
+import ThemeMixin from '@/mixins/theme-mixin';
+
+@Component({
+  components: { AccountDisplay, TagIcon }
+})
+export default class BinancePairsSelector extends Mixins(ThemeMixin) {
+  loading: boolean = false;
+  @Prop({ required: false, type: Boolean, default: false })
+  outlined!: boolean;
+  @Prop({ required: true, type: Array })
+  value!: string[];
+  @Prop({ required: true, type: String })
+  name!: string;
+  @Prop({ required: true, type: String })
+  location!: string;
+  @Emit()
+  input(_value: string[]) {}
+
+  search: string = '';
+  queriedMarkets: string[] = [];
+  selection: string[] = [];
+  allMakerts: string[] = [];
+  multiple: boolean = true;
+
+  async mounted() {
+    this.queriedMarkets = await this.$api.queryBinanceUserMarkets(
+      this.name,
+      this.location
+    );
+    this.allMakerts = await this.$api.queryBinanceMarkets();
```

Comment:
Both these promises should be awaited in a try/catch, otherwise, any network error would blow.

For the loading that @LefterisJP mentioned you could add a `loading: boolean = false` property to the component and then change it to true/false when starting/ending this method, then you can pass that to the autocomplete as `:loading='loading'`